This is my assignment

poojapatel1211 opened the pull request:
My adjective is smart. I plan to add an interactive background as well
as a flashing light bulb. I also want to try to get the bowtie to move.

poojapatel1211 committed:
This is my assignment

My adjective is smart. I plan to add an interactive background as well
as a flashing light bulb. I also want to try to get the bowtie to move.

LadyK commented:
This is a good start. Experiment with motion and time here, to animate this further. What more can happen here to support your adjective?

LadyK commented:
Hi Pooja. The assignment brief outlines weekly pushes demonstrating clear development and progress. I hope it's just a matter of you not pushing your work up to github, but have in fact been putting in the time for this project.

poojapatel1211 committed:
Updated Project 1

I was going to push an updated version of my project on Wednesday but
decided to change my adjective, which is why I haven’t push anything
until now. I decided to change it to paranoid. However, I am having
trouble trying to get the body to repel the mouse.

poojapatel1211 committed:
Final Submission

This is my final project 1. The adjective is skittish.

poojapatel1211 commented:
Uploaded my final submission

poojapatel1211 committed:
Updated Project

I was thinking about the suggestion you made in class last week and
decided to add a roach to the sketch. However, I was unable to get it
to completely replace the mouse.
